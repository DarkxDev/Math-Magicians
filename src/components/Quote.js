const Quote = () => (
  <p className="quote">
    Mathematics is not about numbers, equations,
    computations, or algorithms; it is about understanding.
    <br />
    <br />
    - William Paul Thurston
  </p>
);

export default Quote;
