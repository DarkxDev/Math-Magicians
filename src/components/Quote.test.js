import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quote';

test('Renders correctly', () => {
  const tree = renderer.create(<Quote />);
  expect(tree).toMatchSnapshot();
});
