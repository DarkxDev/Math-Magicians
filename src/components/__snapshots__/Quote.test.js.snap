// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<p
  className="quote"
>
  Mathematics is not about numbers, equations, computations, or algorithms; it is about understanding.
  <br />
  <br />
  - William Paul Thurston
</p>
`;
